github-bot/longhorn_github_bot/__init__.py

```diff
@@ -16,7 +17,11 @@
 FLASK_USERNAME = generate_password_hash(config('flask_username'))
 GITHUB_OWNER = config('github_owner')
 GITHUB_REPOSITORY = config('github_repository')
-ZENHUB_PIPELINE = config('zenhub_pipeline')
+ZENHUB_PIPELINE_ADD_CHECKLIST = config('zenhub_pipeline')
```

Comment:
Suggest changing `zenhub_pipeline` to make it understandable.

Reply:
Refined.